README.md

```diff
@@ -39,6 +39,6 @@ On the chat screen (conversation listing message), instead of sending messages y
 
 `\l`: Go back to conversation list
 
-`\r`: Refresh the current view and fetch latest messages
+`\r`, `(Press Enter with empty string)`: Refresh the current view and fetch latest messages
```

Comment:
As a user, I'd rather expect to read:
```suggestion
`\r`, `(or press <kbd>Enter</kbd> while input is empty)`: Refresh the current view and fetch latest messages
```

What do you think ?

Reply:
👍